object/src/mem.rs

```diff
@@ -100,6 +100,10 @@ pub struct InMemDicomObject<D = StandardDataDictionary> {
     /// It is usually undefined, unless it is part of an item
     /// in a sequence with a specified length in its item header.
     len: Length,
+    /// In case the SpecificCharSet changes we need to mark the object as dirty,
+    /// because changing the character set may change the length in bytes of
+    /// stored text. It has to be public for now because we need
```

Comment:
I suppose it's because we need read access to it in the `tokens` module?

Reply:
Yeah, pretty much. Once we change the charset tag, we need to remember somewhere that that happened. I don't think there is anywhere else where we could potentially store it?